skbio/stats/ordination/_canonical_correspondence_analysis.py

```diff
@@ -9,23 +9,25 @@
 from __future__ import absolute_import, division, print_function
 
 import numpy as np
+import pandas as pd
+from scipy.linalg import svd, lstsq
 
-from ._base import Ordination, OrdinationResults
+from skbio._base import OrdinationResults
 from ._utils import corr, svd_rank, scale
 from skbio.util._decorator import experimental
 
 
-class CCA(Ordination):
```

Comment:
I hate to say this, but under our new API lifecycle, the methods on these classes need to become `deprecated`. I know this commit was probably way in the past and might not be revertible, so if you have to copy it back in from master that is ok.